cmd/kubelet/app/options/options.go

```diff
@@ -60,74 +51,16 @@ type KubeletServer struct {
 
 // NewKubeletServer will create a new KubeletServer with default values.
 func NewKubeletServer() *KubeletServer {
+	config := componentconfig.KubeletConfiguration{}
```

Comment:
as long as the returned `KubeletServer` still contains a defaulted config, it doesn't impact us